react.js

```diff
@@ -1,22 +1,22 @@
 module.exports = {
-  'extends': [
+  extends: [
     './index.js',
     'plugin:jsx-a11y/recommended',
     'plugin:react/recommended',
   ],
 
-  'plugins': [
-    'jsx-a11y',
+  plugins: [
+    // 'jsx-a11y', // TODO: give these rules a full review
```

Comment:
Oops, good catch, I meant to enabled that - why the next order of business will be to add some tests! :)